src/tcas/src/components/TcasComputer.ts

```diff
@@ -1138,6 +1138,15 @@ export class TcasComputer implements TcasComponent {
         this.updateInhibitions();
         this.updateStatusFaults();
         if (this.tcasMode.getVar() === TcasMode.STBY) {
+            this.advisoryState = TcasState.NONE;
+            this.tcasState.setVar(TcasState.NONE);
+            this.correctiveRa.setVar(false);
+            SimVar.SetSimVarValue('L:A32NX_TCAS_STATE', 'Enum', 0);
+            SimVar.SetSimVarValue('L:A32NX_TCAS_RA_CORRECTIVE', 'bool', 0);
```

Comment:
These two lines can be dropped, as they duplicates the lines above (`this.tcasState.setVar`/`this.correctRa.setVar`):
```suggestion
```